_data/current.yml

```diff
@@ -137,6 +137,13 @@
   url: http://saucecon.com/?utm_source=testingconferences
   twitter: saucelabs
   status: Registration is Open
+  
+  - name: SmartBear Connect 2020
```

Comment:
This line has extra spaces. @NoelWurst can you remove this space?  Then the tests will pass.

Reply:
Thx! Deleted those spaces, let's see what that does...